src/styles/font_face.scss

```diff
@@ -1,19 +1,10 @@
-$unicodes: (U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F),
-        (U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116),
-        (U+1F00-1FFF),
-        (U+0370-03FF),
-        (U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB),
-        (U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF),
-        (U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD);
-
-@mixin fontface($font-name, $font-file,$font-weight,$unicode) {
+@mixin fontface($font-name, $font-file,$font-weight) {
   @font-face {
     font-family: $font-name;
     font-style: normal;
     font-weight: $font-weight;
     src: url('../fonts/' + $font-file + '.woff2') format('woff2'),
     url('../fonts/' + $font-file + '.woff') format('woff'),
     url('../fonts/' + $font-file + '.otf') format('opentype');
-    unicode-range: $unicode;
```

Comment:
Can we still have the `unicode-range` property? Instead of a loop, just set it to whatever the unicode range for the latin set is.

Reply:
Okay